<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="8" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="336" started-at="2020-08-08T13:17:24 IST" finished-at="2020-08-08T13:17:24 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="336" started-at="2020-08-08T13:17:24 IST" finished-at="2020-08-08T13:17:24 IST">
      <class name="testNGattributes.TestNGParameters">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:testNGattributes.TestNGParameters@7d907bac]" name="launchBrowser" duration-ms="3" started-at="2020-08-08T13:17:24 IST" finished-at="2020-08-08T13:17:24 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
      </class> <!-- testNGattributes.TestNGParameters -->
      <class name="testNGattributes.TestNGDataProvider">
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String)[pri:0, instance:testNGattributes.TestNGDataProvider@1f28c152]" name="login" duration-ms="21" started-at="2020-08-08T13:17:24 IST" data-provider="getData" finished-at="2020-08-08T13:17:24 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[User1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Password1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String)[pri:0, instance:testNGattributes.TestNGDataProvider@1f28c152]" name="login" duration-ms="5" started-at="2020-08-08T13:17:24 IST" data-provider="getData" finished-at="2020-08-08T13:17:24 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[User2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Password2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String)[pri:0, instance:testNGattributes.TestNGDataProvider@1f28c152]" name="login" duration-ms="11" started-at="2020-08-08T13:17:24 IST" data-provider="getData" finished-at="2020-08-08T13:17:24 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[User3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Password3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- testNGattributes.TestNGDataProvider -->
      <class name="testNGattributes.TestNGAttributes">
        <test-method status="FAIL" signature="login()[pri:0, instance:testNGattributes.TestNGAttributes@7791a895]" name="login" duration-ms="12" started-at="2020-08-08T13:17:24 IST" finished-at="2020-08-08T13:17:24 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [20] but found [10]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [20] but found [10]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at testNGattributes.TestNGAttributes.login(TestNGAttributes.java:12)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="logout()[pri:1, instance:testNGattributes.TestNGAttributes@7791a895]" name="logout" duration-ms="25" started-at="2020-08-08T13:17:24 IST" finished-at="2020-08-08T13:17:24 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- testNGattributes.TestNGAttributes -->
      <class name="testNGattributes.TestNGAttributes2">
        <test-method status="FAIL" signature="login()[pri:0, instance:testNGattributes.TestNGAttributes2@3a5ed7a6]" name="login" duration-ms="8" started-at="2020-08-08T13:17:24 IST" finished-at="2020-08-08T13:17:24 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [20] but found [10]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [20] but found [10]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at testNGattributes.TestNGAttributes2.login(TestNGAttributes2.java:12)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="logout()[pri:0, instance:testNGattributes.TestNGAttributes2@3a5ed7a6]" name="logout" duration-ms="7" started-at="2020-08-08T13:17:24 IST" finished-at="2020-08-08T13:17:24 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- testNGattributes.TestNGAttributes2 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
